File: qqbot-client/src/main/java/com/mieai/qqbot/client/QqMessageSendResult.java
package com.mieai.qqbot.client;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

/** Minimal response fields; QQ may add fields without breaking the sender. */
@JsonIgnoreProperties(ignoreUnknown = true)
public record QqMessageSendResult(
        String id,
        @com.fasterxml.jackson.annotation.JsonProperty("msg_seq") Integer msgSeq,
        String timestamp) {}

File: qqbot-client/src/main/kotlin/com/mieai/qqbot/client/QqMessageSendResult.kt
package com.mieai.qqbot.client

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

/** Minimal response fields; QQ may add fields without breaking the sender. */
@JsonIgnoreProperties(ignoreUnknown = true)
data class QqMessageSendResult(
    val id: String?,
    @JsonProperty("msg_seq") val msgSeq: Int?,
    val timestamp: String?,
)
